lib/entice/logic/ai/seek.ex

```diff
@@ -0,0 +1,91 @@
+defmodule Entice.Logic.Seek do
+  alias Entice.Entity
+  alias Entice.Logic.{Seek, Player.Position, Npc, Movement}
+  alias Geom.Shape.{Path, Vector}
+  alias Geom.Ai.Astar
+
+  defstruct target: nil, aggro_distance: 1000, escape_distance: 2000, path: []
+
+  def register(entity),
+  do: Entity.put_behaviour(entity, Seek.Behaviour, [])
+
+  def register(entity, aggro_distance, escape_distance)
+  when is_integer(aggro_distance) and is_integer(escape_distance),
+  do: Entity.put_behaviour(entity, Seek.Behaviour, %{aggro_distance: aggro_distance, escape_distance: escape_distance})
+
+  def unregister(entity),
+  do: Entity.remove_behaviour(entity, Seek.Behaviour)
+
+  #TODO: Add team attr to determine who should be attacked by whom
+  defmodule Behaviour do
+    use Entice.Entity.Behaviour
+
+    def init(entity, %{aggro_distance: aggro_distance, escape_distance: escape_distance}),
+    do: {:ok, entity |> put_attribute(%Seek{aggro_distance: aggro_distance, escape_distance: escape_distance})}
+
+    def init(entity, _args),
+    do: {:ok, entity |> put_attribute(%Seek{})}
```

Comment:
Here I meant: Do we need this case? Is this actually any use without parameters?